src/main.rs

```diff
@@ -42,8 +45,14 @@ fn main() {
         input = std::fs::read_to_string(args.file).unwrap();
     }
 
+    let bril_args = if args.bril_args.is_empty() {
```

Comment:
Here we either get bril arguments from the command line, and if not we default to the arguments in the bril comment